views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,27 @@
+export const siteTooltipContent = (siteData) => {
+  const tooltipData = []
+  const reserved = 'N/A'
+  const totalLabel = 'Total'
```

Comment:
can we have these live in front-end constants somewhere so we can reference them where needed throughout the front-end?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/82a8267770a707eedb1efe2f1cc9193d667b1d12